src/names/preludes.md

```diff
@@ -18,9 +18,22 @@ There are several different preludes:
 
 ## Standard library prelude
 
-The standard library prelude includes names from the [`std::prelude::v1`]
-module. If the [`no_std` attribute] is used, then it instead uses the names
-from the [`core::prelude::v1`] module.
+Each crate has a standard library prelude, which consists of the names from a single standard library module. The module used depends on the crate's edition, and on whether the [`no_std` attribute] is applied to the crate:
```

Comment:
Can you split sentences with newlines so that each sentence is on a line of its own?

Reply:
Done